src/components/BrowseListingCard.vue

```diff
@@ -93,8 +94,30 @@ export default {
   name: "BrowseListingCard",
   props: ["listings"],
   setup() {
-    function toListingPage(listing_id) {
-      window.location.href = `/admin/listing/${listing_id}`;
+    async function toListingPage(listing_id) {
+      await fetch(`${process.env.SERVER_URL}/modify-statitics/${listing_id}`, {
+        method: "PUT",
+        mode: "cors",
+        credentials: "same-origin",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: {"statistic":"views"},
+      })
+        .then((res) => {
+          if (res.status === 200) {
+            // Need refactor, better way of routing
+            window.location.href = "/admin";
```

Comment:
If we successfully increment the click counter on a listing we want to take them to that listing. I don't have that page created right now, so for now lets not redirect them anywhere and we can come back and change this to the route I create for that page later